nan.h

```diff
@@ -1632,6 +1632,9 @@ class Callback {
     friend class AsyncProgressWorker;
    public:
     // You could do fancy generics with templates here.
```

Comment:
Teeny tiny nit: the comment belongs to the function below, although I'd be fine with removing it, it doesn't add much value.